.github/workflows/ci-build-binary-artifacts.yaml

```diff
@@ -43,8 +41,20 @@ jobs:
       - name: Package Pulsar source
         run: build-support/generate-source-archive.sh
 
+      - uses: docker/setup-buildx-action@v2
+      - run: build-support/copy-deps-versionfile.sh
+
+      - name: Build dependencies Docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: ./pkg/deb
+          tags: ${{ env.TEST_TAG }}
```

Comment:
Could you explain where the `TEST_TAG` environment variable is set?

Reply:
That’s a good point and a good catch. I thought it was auto assigned some unique tag, though it just ends up being an empty tag, so it goes using the default image, which is already on dockerhub. I’ll fix it